Alert and button component

Ash-D23 opened the pull request:
Added Alert and Button Components and Updated Colors and utility classes

Ash-D23 committed:
Update Button Utility Class

Ash-D23 committed:
refactor: updated button class and root color shades

Ash-D23 committed:
refactor: Added Button HTML

Ash-D23 committed:
refactor: Added alert html and updated utility classes

xsaurabhsuthar commented:
Everything looks fine. But, why are you making the floating action button absolute if it's in any of the nested components and the component is using relative then it'll create confusion. Either you can remove absolute or use fixed.

Ash-D23 committed:
fix: button float bug fixed

Ash-D23 commented:
> Everything looks fine. But, why are you making the floating action button absolute if it's in any of the nested components and the component is using relative then it'll create confusion. Either you can remove absolute or use fixed.

Fixed the bug, changed button position to fixed, Thank you
